index.rst

```diff
@@ -77,19 +85,39 @@ categorized as:
 Tag Fragments
 -------------
 
-In general, tags can be postfixed with an optional Cycle number or a
-``rest`` set of one-or-more underscore-separated fields.  The Cycle is
-currently unique to T&S RSP instances, and specifies the version of XML
-used in this image.  It will have the form ``c<digits>.<digits>`` where
-the first group of digits is the cycle number and the second group of
-digits is the build iteration for that cycle, e.g. ``c0019.001``.
+Primary
+^^^^^^^
+The primary version of a tag is derived from (and in most cases identical to) the EUPS tag of the underlying DM stack included in the image.
+It is always the first component of the tag.
+
+RSP Build Counter
+^^^^^^^^^^^^^^^^^
+
+Tags may include an identifier specifying an RSP image build count. 
+In general, any run of the GitHub Action that builds a sciplat-lab image will increment this counter.
+It is currently in the 2000s, and we do not anticipate it exceeding 10,000 during the lifetime of the project.
+We anticipate only including this counter in Release, Release Candidate, and Experimental builds derived from one of the former two categories, although it will continue to increment with each build regardless of type.
```

Comment:
"Anticipate" introduces a lot of ambiguity about what we are going to do and when we're going to do it. I think this should just say directly what we're going to do, in the present tense since we can merge all of this work close together. If we change what we do, we should change the technote.